go.sum

```diff
@@ -1,26 +1,54 @@
-github.com/aws/aws-sdk-go v1.25.19 h1:sp3xP91qIAVhWufyn9qM6Zhhn6kX06WJQcmhRj7QTXc=
-github.com/aws/aws-sdk-go v1.25.19/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
+github.com/aws/aws-sdk-go-v2 v1.36.3 h1:mJoei2CxPutQVxaATCzDUjcZEjVRdpsiiXi2o38yqWM=
```

Comment:
If we haven't already, can we use this opportunity to bump all deps, and check that tests pass?

If they don't, let's keep this one to just AWS. If they do still pass, no harm updating!